eclair-core/src/main/scala/fr/acinq/eclair/NodeParams.scala

```diff
@@ -246,7 +246,10 @@ object NodeParams {
         searchMaxRouteLength = config.getInt("router.path-finding.max-route-length"),
         searchMaxCltv = config.getInt("router.path-finding.max-cltv"),
         searchMaxFeeBaseSat = config.getLong("router.path-finding.fee-threshold-sat"),
-        searchMaxFeePct = config.getDouble("router.path-finding.max-fee-pct")
+        searchMaxFeePct = config.getDouble("router.path-finding.max-fee-pct"),
+        searchRatioCltv = config.getDouble("router.path-finding.ratio-cltv"),
```

Comment:
We should check that configured ratios are valid

Reply:
Done in a0caf94